.gitignore

```diff
@@ -30,5 +29,7 @@ bin
 .vscode
 *~
 
+# Modelmesh development related artifacts
 devbuild
 .develop_image_name
+.bash_history
```

Comment:
Shouldn't .bash_history be in the parent folder, rather than in devbuild?

Reply:
Yup, just organizing ModelMesh development files and artifacts in one section :-)